cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -461,14 +463,14 @@ private PodInfo createPodInfo(Map<String, String> conf) throws IOException, ApiE
       .findFirst()
       .orElse(pod.getSpec().getContainers().get(0));
 
+    Set<String> configs = ImmutableSet.of(CDAP_CONFIG_NAME, HADOOP_CONFIG_NAME, SECURITY_CONFIG_NAME);
```

Comment:
nit: why not to reuse CONFIG_NAMES and add CDAP_CONFIG_NAME, HADOOP_CONFIG_NAME and SECURITY_CONFIG_NAME to it.

Reply:
Done